dungeon/src/starter/Starter.java

```diff
@@ -133,12 +133,15 @@ public static void main(String[] args) {
     } catch (NullPointerException | InterruptedException | InvocationTargetException e) {
       // no configuration, so let's abort here
       System.err.println("No .dng file selected. Exiting ...");
+      throw new RuntimeException(e);
```

Comment:
@tgrothe was genau soll das hier werden? wir sind dabei, das spiel zu beenden. was genau soll da noch eine neue rte?!

Reply:
hmmm??? was genau meinst du? 

wenn du dich darauf beziehst, dass man mit der message in der exception arbeiten könnte, dann wäre das hier die **ausgabe** der message, falls vorhanden. aber doch nicht das werfen einer neuen exception? zumal das `main()` hier dann eh zu ende ist?!